python-package/lightgbm/basic.py

```diff
@@ -22,7 +22,7 @@ def _load_lib():
     """Load LightGBM Library."""
     lib_path = find_lib_path()
     if len(lib_path) == 0:
-        raise Exception("cannot find LightGBM library")
+        return None
```

Comment:
I don't think so, because the exception should be raised at https://github.com/Microsoft/LightGBM/blob/master/python-package/lightgbm/libpath.py#L29.